### PR TITLE
Watermark Maker(For Images)

### LINKED ISSUE #546: Watermark on Image
AIM
 Watermark on the image using text

WHAT IT WILL DO
Given an image and text as an input to display a watermark on the given image.

### PR DESCRIPTION
**Pull request by aishwaryachand**
# Description

- Apart from the previous PR #628  Added user defined position option and textwrap  of multiple lines 

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [x] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by santushtisharma10 — APPROVED** [review]
@aishwaryachand 💯💯

**Comment by santushtisharma10** [comment]
@Kushal997-das

File: Watermark Maker(On Image)/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+#### WITH WATERMARK
+[![Test-Image2.png](https://i.postimg.cc/LsDcBY73/Test-Image2.png)](https://postimg.cc/Z99sJRJW)
+
+[![Centre-Text.png](https://i.postimg.cc/X7Wthgbk/Centre-Text.png)](https://postimg.cc/MXssfVXn)
+
+
+## Author
+Aishwarya Chand @aishwaryachand
+
```
Comment (kaustubhgupta) [inline Watermark Maker(On Image)/README.md:27]: Remove this longgg extra space

File: Watermark Maker(On Image)/Watermarker(Image).py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+path = path.strip('""')
+im = Image.open(path)
+width, height = im.size
+
+#Taking input for the text
+text = input('Enter the text for the  watermark:' )
+font = ImageFont.truetype('arial.ttf', 20 )
+
+#Taking Input For the Position of the text
+print("Where you want to input the text . Pls enter\n 1 - Top Left \n 2 - Top Right \n 3 - Bottom Left \n 4 - Bottom Right \n 5 - Center")
```
Comment (kaustubhgupta) [inline Watermark Maker(On Image)/Watermarker(Image).py:15]: Here, the choice is ambiguous. One is not sure whether to input is the number associated with position or the position itself. Add a print statement to clarify this

File: Watermark Maker(On Image)/Watermarker(Image).py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+from PIL import Image, ImageDraw, ImageFont
+
+#Taking Input From user
+path = input("Input the path of the image: ")
+path = path.strip('""')
+im = Image.open(path)
+width, height = im.size
+
+#Taking input for the text
+text = input('Enter the text for the  watermark:' )
```
Comment (kaustubhgupta) [inline Watermark Maker(On Image)/Watermarker(Image).py:11]:
```suggestion
text = input('Enter the text for the  watermark: ' )

```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by antrikshmisri** [comment]
@kaustubhgupta Almost all scripts don't follow PEP8 requirements. I think deploying a autopep8 bot on the repo would save us a ton of time. [Here's](https://github.com/OrkoHunter/pep8speaks) the link to it

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta Almost all scripts don't follow PEP8 requirements. I think deploying a autopep8 bot on the repo would save us a ton of time. [Here's](https://github.com/OrkoHunter/pep8speaks) the link to it

Yes, this is a good idea. @avinashkranjan can you include this in workflows or should we make an issue for the participants to contribute?

**Comment by aishwaryachand** [comment]
Made the changes as requested. Kindly review .

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

File: Watermark Maker(On Image)/README.md (hunk 1 above)
Comment (kaustubhgupta) [reply Watermark Maker(On Image)/README.md:25]: @aishwaryachand This is still not removed

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Review by kaustubhgupta — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@antrikshmisri

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..💯

**Pull request merged**